README.md

```diff
@@ -99,7 +99,7 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
 
-- **[Person class]**
+- **[Person class with Decorators]**
 - **[Student class]**
 - **[Teacher class]**
```

Comment:
- [OPTIONAL] It will be nice to update your getting started section with instructions on how to clone and run your project locally.

Reply:
This is noted